src/Sarif.Viewer.VisualStudio/Resources.resx

```diff
@@ -165,6 +169,18 @@
     <value>The log file '{0}' was not found.</value>
     <comment>{0} will be the full file path</comment>
   </data>
+  <data name="ProcessingLogFileFormat" xml:space="preserve">
+    <value>Processing SARIF log {0}.</value>
+    <comment>Used to display processing log file information in VS's task center UI.</comment>
+  </data>
+  <data name="ProcessLogFiles" xml:space="preserve">
+    <value>Processing SARIF logs.</value>
+    <comment>Used to display processing log file information in VS's task center UI.</comment>
+  </data>
+  <data name="ProcessLogFilesComplete" xml:space="preserve">
+    <value>Processing SARIF logs completed.</value>
```

Comment:
>Processing SARIF logs completed. [](start = 11, length = 32)

"Done processing SARIF logs."

Reply:
Super-nit but I'd prefer "Done" to "Completed"

---
In reply to: [494523828](https://github.com/microsoft/sarif-visualstudio-extension/pull/202#discussion_r494523828) [](ancestors = 494523828)